core-blocks/embed/index.js

```diff
@@ -109,27 +128,53 @@ function getEmbedBlockSettings( { title, description, icon, category = 'embed',
 				const { url } = this.props.attributes;
 				const { setAttributes } = this.props;
 
+				if ( undefined === url ) {
```

Comment:
Is the condition in `componentWillMount` redundant now? For that matter, is the `setState` redundant as well?

Reply:
Ah, good catch! Removed!